### DISCUSSION TITLE
Events timestamps in localtime

### DISCUSSION THREAD

**Discussion by kpine** [zwave-js/zwave-js-ui#3517] [Q&A] [answered]
The timestamp of events in the node Events tab is UTC, but ZUI may be configured for a local timezone. This makes it harder to correlate events with local wall clock time, and with the logs which I have configured for localtime. Would it be possible to show these timestamps using the configured timezone? It is currently is hardcoded to UTC.

https://github.com/zwave-js/zwave-js-ui/blob/e7ce406bbb2c742b2bc6af0d6c70bfc0a7b548d3/src/components/nodes-table/ExpandedNode.vue#L253-L257

Also, it would be nice if events could be reverse sorted. I'm usually just interested in the latest events, which means I need to scroll past a hundred events to the bottom of the pane just to see the latest ones.

**Comment by robertsLando** [comment, marked as answer]
> Would it be possible to show these timestamps using the configured timezone

You mean the OS timezone, right?

> Also, it would be nice if events could be reverse sorted. I'm usually just interested in the latest events, which means I need to scroll past a hundred events to the bottom of the pane just to see the latest ones.

No problem for this

**Comment by kpine** [comment]
> > Would it be possible to show these timestamps using the configured timezone
> 
> You mean the OS timezone, right?

Correct, I am configuring my docker container using the `TZ` variable for the local timezone.

> 
> > Also, it would be nice if events could be reverse sorted. I'm usually just interested in the latest events, which means I need to scroll past a hundred events to the bottom of the pane just to see the latest ones.
> 
> No problem for this

:+1: Shall I create a proper enhancement request?

**Reply by robertsLando** [reply]
Yes please :)
